export_database.py

```diff
@@ -0,0 +1,149 @@
+from pymongo import MongoClient
+from pymongo import errors
+import bson
+from bson.json_util import dumps, loads
+from os import environ as env
+import sys
+import argparse
+
+old_mongodb_uri = ""
+new_mongodb_uri = ""
+
+def export_db():
+    '''
+        Tries to Exports the whole Database (each and every notice)
+        Saves notices already in target Database in repeated_notices.bson
+        Saves notices failed to be inserted in defaulters.bson
+
+        Can be run multiple times with no fear of multiple entries in case of any confusion
+    '''
+    print("connecting to old DB")
+    mc_old = MongoClient(old_mongodb_uri)
+    print("collecting notices cursor from old database")
+    notices_cursor = mc_old.get_default_database().notices.find()
+    print("connecting to new DB")
+    mc_new = MongoClient(new_mongodb_uri)
+    print("connected to new DB")
+
+    defaulters = []
+    repeated_notices = []
+    for notice in notices_cursor:
+        try:
+            mc_new.get_default_database().notices.insert(notice)
+            print("inserted notice: ", notice)
+        except errors.DuplicateKeyError:
+            print("entry already in database")
+            repeated_notices.append(notice)
+        except Exception as ex:
+            print("error in inserting: ", ex)
+            defaulters.append(notice)
+
+    if(len(defaulters)):
+        print("\nDefaulters Present!!!")
+    else:
+        print("\nYohoo!! No Defaulters")
+
+
+    print("Saving defaulters and repeated notices...")
+    with open("defaulters.bson", "w") as f:
+        f.write(dumps(defaulters))
+        f.close()
```

Comment:
In case of using with, `f.close` is not required as the file is closed automatically when the `with` context dies. So, may be you'd like to remove this line.